nbs/api/14_quarto.ipynb

```diff
@@ -11,6 +11,14 @@
     "- order: 14"
```

Comment:
<p>I'd remove this - it takes up a lot of room in the docs and dupes what can be found in the link you provide above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>